sdk/cosmosdb/cosmos/test/public/functional/container.spec.ts

```diff
@@ -406,6 +454,36 @@ describe("createIfNotExists", function () {
     const { resource: readDef } = await container.read();
     assert.equal(def.id, readDef.id);
   });
+
+  it("should handle container does not exist - hierarchical partitions", async function () {
```

Comment:
@v1k1 i am not able to understand significance of these 2 tests, can you explain?